UI/window-basic-settings-stream.cpp

```diff
@@ -1177,3 +1182,241 @@ bool OBSBasicSettings::IsServiceOutputHasNetworkFeatures()
 
 	return false;
 }
+
+static bool service_supports_codec(const char **codecs, const char *codec)
+{
+	if (!codecs)
+		return true;
+
+	while (*codecs) {
+		if (astrcmpi(*codecs, codec) == 0)
```

Comment:
Is there a reason this is a case-insensitive compare? Feels like we should `strcmp` here since this data shouldn't be coming from a user and should be all-lowercase.